core.js

```diff
@@ -874,6 +874,24 @@ global._runHtml = (element, window) => {
               });
           }
         }
+      } else if (el instanceof window.HTMLLinkElement) {
+        if (el.run()) {
```

Comment:
Perhaps this logic can be part of the class as an internal method? Also a comment of what this code does could be useful in the future.

Reply:
Agree with both.

Not actually related to this PR though, so will make it separate.